app/src/main/java/database/CrimeDao.kt

```diff
@@ -0,0 +1,17 @@
+package database
+
+import androidx.lifecycle.LiveData
+import androidx.room.Dao
+import androidx.room.Query
+import com.bignerdranch.android.criminalintent.Crime
+import java.util.*
+
+@Dao
+interface CrimeDao {
+
+    @Query("SELECT * FROM crime")
+    fun getCrimes(): LiveData<List<Crime>>
+
+    @Query("SELECT * FROM crime WHERE id=(:id)")
+    fun getCrime(id: UUID): LiveData<Crime?>
```

Comment:
*nit pick*

If you provide an `id` as an argument here, then for the sake of consistency you could consider renaming this method (and any other that ahs the signature of `getCrime(id: String)`) to `getCrimeById`

Reply:
Will keep it in mind